nixos/modules/services/databases/mysql.nix

```diff
@@ -357,21 +358,17 @@ in
         preStart = if isMariaDB then ''
           if ! test -e ${cfg.dataDir}/mysql; then
             ${mysql}/bin/mysql_install_db --defaults-file=/etc/my.cnf ${mysqldOptions}
-            touch /tmp/mysql_init
+            touch ${cfg.dataDir}/mysql_init
           fi
         '' else ''
           if ! test -e ${cfg.dataDir}/mysql; then
             ${mysql}/bin/mysqld --defaults-file=/etc/my.cnf ${mysqldOptions} --initialize-insecure
-            touch /tmp/mysql_init
+            touch ${cfg.dataDir}/mysql_init
```

Comment:
@Izorkin maybe we still can use `/tmp`... https://github.com/NixOS/rfcs/pull/59#discussion_r440911801

Reply:
Don't think that will work.

Reply:
I don't like fixed paths in `/tmp`. They are vulnerable to security issues.

Reply:
That is what `PrivateTmp` is for. But I'm good either way :+1: